libdnf/hy-goal.c

```diff
@@ -46,6 +46,7 @@
 #include "hy-selector-private.h"
 #include "hy-util.h"
 
+#define BLOCK_SIZE 15
```

Comment:
Why is the the `BLOCK_SIZE` set to `15`? Please explain this in a comment in the sources.